dotcom-rendering/src/model/buildLightboxImages.ts

```diff
@@ -10,43 +8,19 @@ import type {
 } from '../types/content';
 import { isCartoon, isImage } from './enhance-images';
 
+/** Used to determine if a lightbox can be created */
+const THRESHOLD = 620;
+
 /**
- * Only allow the lightbox to show images that have a source with a width greater
- * than a threshold.
+ * Only allow the lightbox to show images whose largest source
+ * is higher than a specific threshold.
  *
  * Frontend has similar logic here:
  * https://github.com/guardian/frontend/blob/126dfcbc1aa961650b7f7ff41ee50a12782bb62e/common/app/model/content.scala#L549
  *
  */
-const isLightboxable = (
-	element: ImageBlockElement | CartoonBlockElement,
-	orientation: Orientation,
-): boolean => {
-	switch (orientation) {
-		case 'landscape':
-			if (
-				element._type ===
-				'model.dotcomrendering.pageElements.CartoonBlockElement'
-			) {
-				return element.variants.some((v) =>
-					v.images.some(isWideEnough),
-				);
-			} else {
-				return element.media.allImages.some(isWideEnough);
-			}
-		case 'portrait':
-			if (
-				element._type ===
-				'model.dotcomrendering.pageElements.CartoonBlockElement'
-			) {
-				return element.variants.some((v) =>
-					v.images.some(isHighEnough),
-				);
-			} else {
-				return element.media.allImages.some(isHighEnough);
-			}
-	}
-};
+const isLightboxable = (width: number, height: number): boolean =>
+	Math.max(width, height) > THRESHOLD;
```

Comment:
Knowing the orientation is unnecessary, as we only consider the largest dimension.